hardware/codebase/up4000/aveiro/device_config.json

```diff
@@ -27,7 +27,7 @@
   "billDispenser": {
     "model": "gsr50",
     "cassettes": "1",
-    "stackers": "3"
+    "recyclers": "6"
```

Comment:
This field and the above may be numbers, `parseInt(n) = n` when `n` is a number.